tests/publiccloud/migration.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright 2016-2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Basic DMS test
+#
+#   This test does the following
+#    - Installs SLE15-Migration and sle15-activation
+#    - Reboot the system
+#    - Check the system version is the expected one
+#
+# Maintainer: Jesus Bermudez Velazquez <jesus.bv@suse.com>
+
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+
+our $target_version = get_required_var('TARGET_VERSION');
+
+sub run {
+    my ($self, $args) = @_;
+    $self->select_serial_terminal;
+    my $provider = $self->provider_factory();
+    my $instance = $provider->create_instance();
+    $instance->wait_for_guestregister();
+
+    record_info('INFO', $target_version);
+    record_info('INFO', get_var('TARGET_VERSION'));
+
+    sleep 90;    # wait for a bit for zypper to be available
+
+    record_info('installs SLE15-Migration and suse-migration-sle15-activation');
+
+    $instance->run_ssh_command(cmd => "sudo zypper in -y SLES15-Migration suse-migration-sle15-activation");
```

Comment:
```suggestion
    $instance->run_ssh_command(cmd => "sudo zypper -n in SLES15-Migration suse-migration-sle15-activation");
```

Reply:
https://suse.slack.com/archives/C02CGKBCGT1/p1649088732608329

Reply:
No, this particular test is for PAYG